index.html

```diff
@@ -20,6 +20,11 @@
       <h1>Owatonna Even/Odd Parking</h1>
     </div>
 
+    <div class="spacer">
+      <div class="line"></div>
+      <img class="lights" src="/assets/lights_stiched-1.png" style="display: none">
+    </div>
```

Comment:
I was hoping this could be done purely in CSS, but I could not figure out how to get it working. I could sort of get it working using various `border-image` things, but it was pretty sketchy.

Reply:
(I say this just because all of the code and layout would be way more straightforward if CSS could help out a little more. 😿)

Reply:
I initially though of using CSS, but I thought it might end up being more simple to use JavaScript.

Reply:
Nice! Yeah, in this case the CSS was very finicky, and I don't know if it is even possible. The JS was then way quicker to implement.